added info about missing TLS 1.3 support (#1001)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/technical-reference-details-about-encryption.md

    
- TLS version 1.0 (TLS 1.0)
    

>  TLS version 1.3 (TLS 1.3) is not supported by Office 365 yet. TLS 1.0 and TLS 1.1 support will be deprecated October 31, 2018. See [Deprecating support for TLS 1.0 and 1.1 and what this means for you](technical-reference-details-about-encryption.md#TLS11and12deprecation) for more information. 

Reviewer: ```suggestion
 TLS version 1.3 (TLS 1.3) is not supported by Office 365 yet. Since October 31, 2018, support for TLS 1.0 and TLS 1.1 has been deprecated. See [Deprecating support for TLS 1.0 and 1.1 and what this means for you](technical-reference-details-about-encryption.md#TLS11and12deprecation) for more information. 
```
